app/views/exclusion/WhatNextRescind.scala.html

```diff
@@ -32,49 +32,47 @@
         empRef: EmpRef
 )(implicit request:Request[_], messages: Messages)
 
-@govukLayoutWrapper(pageTitle = messages("whatNext.rescind.heading") + " - " + messages("Service.title"), showBackLink = false,  empRef =Some(empRef.toString)) {
+@govukLayoutWrapper(pageTitle = messages("whatNext.rescind.heading") + " - " + messages("Service.title"), showBackLink = true,  empRef =Some(empRef.toString)) {
 
     <section id="print-section">
         <div id="printContainer"></div>
     </section>
-
     <div id="confirmation" class="govuk-panel govuk-panel--confirmation">
         <h1 id="title" class="govuk-panel__title">
             @messages("whatNext.rescind.heading")
         </h1>
-        <p id="confirmation-p">
-            @messages("ExclusionRemoval.WhatNext.lede", name, messages("BenefitInKind.label." + uriInformation.iabdValueURLDeMapper(iabdType)), taxYearRange.cy+"")
-        </p>
     </div>
 
-    <h2 class="govuk-heading-m">
-        @messages("whatNext.subHeading")
-    </h2>
-    <div class="govuk-form-group" >
-        <ul class="govuk-list govuk-list--bullet">
-            <li>
-                @Html(messages("ExclusionRescind.WhatNext.p1"))
-            </li>
-
-            <li>
-                @Html(messages("ExclusionRescind.WhatNext.p2"))
-            </li>
+    <div class="govuk-form-group">
+        <p class="govuk-body" id="confirmation-p">
+            @messages("ExclusionRemoval.WhatNext.p1", name, messages("BenefitInKind.label." + uriInformation.iabdValueURLDeMapper(iabdType)), taxYearRange.cy+"" )
+        </p>
+        <h2 class="govuk-heading-m">
+            @messages("ExclusionRemoval.whatYouNeedToKnow.subHeading")
+        </h2>
+        <p class="govuk-body">@messages("ExclusionRemoval.whatYouNeedToKnow.p1", taxYearRange.cy+"")</p>
```

Comment:
what is the use of +""

Reply:
removed it and re-run the testing looks fine.